lintly/parsers.py

```diff
@@ -9,7 +9,6 @@
 
 from .violations import Violation
 
-
 class BaseLintParser(object):
```

Comment:
flake8 generally requires two empty lines before a new class. Any idea why the second empty line was deleted?